### PR TITLE
[incubator/elastic-stack] Add fluentd-elasticsearch and elasticsearch-exporter to elastic-stack

### PR DESCRIPTION
**Pull request by rendhalver**
Update repository for elasticseach and elasticsearch-curator

Signed-off-by: Pete Brown <pete.brown@powerhrg.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

File: incubator/elastic-stack/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ description: A Helm chart for ELK
 home: https://www.elastic.co/products
 icon: https://www.elastic.co/assets/bltb35193323e8f1770/logo-elastic-stack-lt.svg
 name: elastic-stack
-version: 0.9.2
+version: 0.9.3
```
Comment (desaintmartin) [inline incubator/elastic-stack/Chart.yaml]: Maybe bump minor version since it's new feature?

**Review by desaintmartin — COMMENTED** [review]
LGTM except regarding version, although I would prefer exporter to be bundled as a sidecar container of ES container itself (less configuration burden, less potential errors)

File: incubator/elastic-stack/Chart.yaml (hunk 1 above)
Comment (rendhalver) [reply incubator/elastic-stack/Chart.yaml]: Good point.

**Comment by rendhalver** [comment]
That would be a limitation of the exporter chart.
Let's fix it there.

**Pull request merged**